OrbitGl/DataManager.h

```diff
@@ -5,6 +5,8 @@
 #ifndef ORBIT_GL_DATA_MANAGER_H_
 #define ORBIT_GL_DATA_MANAGER_H_
 
+#include <CallstackTypes.h>
```

Comment:
and this include won't be needed anymore.

Reply:
done.